forte/data/converter/converter.py

```diff
@@ -277,7 +277,14 @@ def convert(
         # Convert to target type
         if not self.to_numpy and not self.to_torch:
             return data_list, masks_list
-
+        try:
```

Comment:
we should put these try blocks at the beginning of the function. If this fails, there is no need to execute the rest.

Reply:
you probably want to do something like

```
if self.to_torch
   try import
```

at the beginning of the function